android/assets/jsons/Civ V - Gods & Kings/Units.json

```diff
@@ -1610,6 +1613,7 @@
 		"unitType": "Civilian",
 		"uniques": ["Can speed up construction of a building",
             "Can instantly construct a [Manufactory] improvement <by consuming this unit>",
+            "Is part of Great Person group [Great Person]",
```

Comment:
I think it matches what's expected now. What do you think we should call this Artist, Merchant, Scientist and Engineer group?

Reply:
It has not been decide yet, so this below is normal?

![image](https://github.com/user-attachments/assets/117a9dbe-5fa1-4616-8287-c45dd1729442)

Reply:
That's correct, but I'm still not convinced `Great Person` is the best group name to use here. Perhaps we use `pool` so the text says...

> Is part of Great Person group pool

?

Reply:
`Is part of Great Person group: Expert`  (Artist / Scientist / Merchant / Engineer = Expert)
?